app/models/order.rb

```diff
@@ -11,4 +11,11 @@ def format_updated_at
     updated_at.strftime("%B %e, %Y")
   end
 
+  def subtotal(escape_id, quantity)
+    sprintf('%.2f', (Escape.find(escape_id).price * quantity))
```

Comment:
We do this decimal conversion in a couple different places throughout this project. At some point, should we try to DRY this out?